stylelint.config.js

```diff
@@ -43,7 +44,7 @@ module.exports = {
     'number-no-trailing-zeros': true,
     'property-no-unknown': null,
     'property-no-vendor-prefix': null,
-    'scss/at-import-no-partial-leading-underscore': true,
+    'scss/load-no-partial-leading-underscore': true,
```

Comment:
Rule name had changed in new versions of stylelint. (Same for `unit-allowed-list`)